api/sixtyfps-cpp/tests/datastructures.cpp

```diff
@@ -72,3 +73,24 @@ TEST_CASE("Property Tracker")
     REQUIRE(!tracker1.is_dirty());
 }
 
+TEST_CASE("Image")
+{
+    using namespace sixtyfps;
+
+    // ensure a backend exists, using private api
+    private_api::ComponentWindow wnd;
```

Comment:
Maybe better to call `sixtyfps::testing::init();`

Reply:
Good idea!

Reply:
Looks like this will require adding the feature support to corrosion, in order to enable the testing feature.